components/Modals/GroupNotifyMeModal/NotifyMeForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { useModalDispatch, showModal } from 'providers/ModalProvider';
+import { useCampuses, useCurrentUser } from 'hooks';
+
+import { Box, Button, FormLabel, Select } from 'ui-kit';
+
+function CampusSelect(props = {}) {
+  const { loading, campuses } = useCampuses();
+
+  return (
+    <>
+      <FormLabel>Select a Campus</FormLabel>
+      <Select
+        id="campus"
+        name="campus"
+        defaultValue={props.selectedCampusId}
+        onChange={props.onChange}
+        disabled={loading}
+      >
+        <Select.Option value="">Select...</Select.Option>
+        {campuses?.map(item => (
+          <Select.Option key={item.id} value={item.id}>
+            {item.name}
+          </Select.Option>
+        ))}
+      </Select>
+    </>
+  );
+}
+
+function NotifyMeForm(props = {}) {
+  const { currentUser } = useCurrentUser();
+  const [campusId, setCampusId] = useState(props.initialCampusId || '');
+  const modalDispatch = useModalDispatch();
+
+  const handleCampusSelect = event => {
+    setCampusId(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    console.group('%cNotify Me form submitted', 'color: limegreen');
```

Comment:
Probably could kill these console.logs now

Reply:
I can remove them, but this is the "handoff" point so purposely left them in for testing validation.
I'll leave them in, but commented out, until the final code is in place.

Reply:
Updated in 5e40fa6

Reply:
O, okay 👍 Either way is good for me.